.github/workflows/tests.yml

```diff
@@ -71,7 +71,7 @@ jobs:
   # - package-loock.json does not exist
   test-yarn-success:
     runs-on: ubuntu-latest
-    name: Test successful npm check
+    name: Test successful yarn check
```

Comment:
Noticed this copy/paste whoopsie, so I fixed it in this branch.

Reply:
😱